mpf/config_players/device_config_player.py

```diff
@@ -105,8 +105,8 @@ def _expand_device(self, device):
                 if not self.__class__.allow_placeholders_in_keys or "(" not in device_name:
                     # no placeholders
                     return self.raise_config_error(
-                        "Could not find a {} device with name or tag {}.".format(
-                            self.device_collection.name, device_name),
+                        "Could not find a {} device with name or tag {}, from list {}".format(
```

Comment:
When the device list has an empty option, the `device_name` is empty and this error log isn't very helpful. Adding `device_or_tag_names` helps to identify the error.